codeforamerica/vita-min
# Allow editing of Signature Method for drop-off clients

## [GYR1-135](https://codeforamerica.atlassian.net/browse/GYR1-135)
## What was done?
 * `signature_method` field was added to `update_client_form`
 * `tax_info` partial was added to edit.html.erb form including signature_method
![image](https://github.com/codeforamerica/vita-min/assets/17094895/bb8736a2-b2c3-439c-a63f-9e4e93a9dbb9)
 * Specs were updated to cover the changes
## How to test?
 * Log into the hub and edit a client. You should see a new "Tax Info" section at the bottom before the save button. Changes to this field should be persisted on save, and visible in the show view
![image](https://github.com/codeforamerica/vita-min/assets/17094895/319e2893-2d8a-4d85-a9d8-66336d59f8a0)


[GYR1-135]: https://codeforamerica.atlassian.net/browse/GYR1-135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
